frontend/app/views/fragments/bundle/bundleOfferingsB.scala.html

```diff
@@ -1,3 +1,196 @@
 @import abtests.BundleVariant
+@import views.support.Asset
 
 @(bundleVariant: BundleVariant)
+<section class="bundle-section">
+    <div class="bundle-offering-b l-constrained">
+        <div class="bundle-offering-b__subscribe">
```

Comment:
I'm not sure I agree with wrapping all the packages in this enclosing `div`. It feels like this div should end after the "Enjoy a richer Guardian experience" paragraph.